retranslate/.babelrc

```diff
@@ -0,0 +1,12 @@
+{
+  "plugins": [
+    "syntax-class-properties",
+    "syntax-object-rest-spread",
+    "syntax-jsx",
+    "syntax-flow",
+    ["./retranslate/babel-plugin-retranslate/dist-plugin.js", {
+      "replacement": "../public/i18n/de.txt"
```

Comment:
I guess this should be `en_US.txt`?

Reply:
Will fix, and add script to package json